src/filesinkhelper.ipp

```diff
@@ -85,9 +85,9 @@ namespace g3 {
          return ss_entry.str();
       }
 
-      std::string createLogFileName(const std::string &verified_prefix) {
+      std::string createLogFileName(const std::string &verified_prefix, const std::string &logger_id=".g3log.") {
          std::stringstream oss_name;
-         oss_name << verified_prefix << ".g3log.";
+         oss_name << verified_prefix << logger_id;
```

Comment:
I'm OK with the unzip. Otherwise there's a www connection dependency.

I think a good and descriptive error message and CMake abort could be added to make it obvious. 

There's also a Travis build script that builds and executes all the tests. That might be cool to convert for non-Travis usage and add to *nix and windows